File: frontend-playstore/PusulaService/app/src/main/java/com/pusula/service/data/model/SettingsModels.kt
package com.pusula.service.data.model

data class UserDTO(
    val id: Long? = null,
    val username: String,
    val fullName: String? = null,
    val role: String,
    val password: String? = null
)

data class VehicleDTO(
    val id: Long? = null,
    val companyId: Long? = null,
    val licensePlate: String,
    val driverName: String? = null,
    val isActive: Boolean = true
)

data class CompanyDTO(
    val id: Long? = null,
    val name: String,
    val phone: String? = null,
    val address: String? = null,
    val email: String? = null,
    val logoUrl: String? = null
)

File: frontend-appstore/PusulaService/Models/SettingsModels.swift
import Foundation

struct UserDTO: Codable, Identifiable {
    var id: Int?
    let username: String
    let fullName: String?
    let role: String
    var password: String?
}

struct VehicleDTO: Codable, Identifiable {
    var id: Int?
    let companyId: Int?
    let licensePlate: String
    let driverName: String?
    var isActive: Bool?
}

struct CompanyDTO: Codable, Identifiable {
    var id: Int?
    let name: String
    let phone: String?
    let address: String?
    let email: String?
    let logoUrl: String?
}

struct ResetPasswordRequest: Codable {
    let newPassword: String
}
